python/test/test_api.py

```diff
@@ -137,6 +137,7 @@ def test_memory_fallback_to_cpu(self, server_options):
 
         tritonserver.default_memory_allocators[tritonserver.MemoryType.GPU] = allocator
```

Comment:
should this be removed - or changed in some way to indicate the allocator is internal ....

Reply:
yes, updated the test to indicate the allocator is internal, and it will always use CPU memory regardless of the backend memory preference.